src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,10 +1792,11 @@ static void logEx(final String... lines) {
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
 
-	public static Message SKRIPT_PREFIX = new Message("skript.prefix");
+	private static final String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
+	private static final Message SKRIPT_PREFIX_MESSAGE = new Message("skript.prefix");
 
 	public static void info(final CommandSender sender, final String info) {
-		sender.sendMessage(Utils.replaceEnglishChatStyles(SKRIPT_PREFIX + info));
+		sender.sendMessage(Utils.replaceEnglishChatStyles((SKRIPT_PREFIX_MESSAGE.toString().isEmpty() ? SKRIPT_PREFIX : SKRIPT_PREFIX_MESSAGE) + info));
```

Comment:
Hmm maybe it'd be a good idea to add a getValueOrDefault method to Message instead of doing this

Reply:
I'll see what I can do shouldn't be too hard right?

Reply:
Let me know any changes you'd like instead of what I went with for this, everything should be sorted now